lib/testing/README.md

```diff
@@ -23,7 +23,7 @@ or
 `./scripts/test-e2e-lib.sh -host http://myadf.example.com -f process-services-cloud`
 
 ### How can I run the *cloud* e2e against a *remote* env with *full chrome* ?
-Add to `.env.cloud` and run `./scripts/test-e2e-lib.sh`
+Add to `.env.cloud` and run `./scripts/test-e2e-lib.sh -env .env.cloud`
```

Comment:
we need to specify the path right here, I am only able to run the tests on this branch with this command =>
./scripts/test-e2e-lib.sh -env ./e2e/.env.cloud
so apecifying e2e, in the path is necessary, I tihnk.

Reply:
or we can chage 
  `require('dotenv').config({path: ENV_FILE});`
to 
  `require('dotenv').config({path: './e2e'/ENV_FILE});`

Reply:
in this example we don't specify in which folder you have the env file is not important to have it in e2e you can have it where you like more